cmd/can/main.go

```diff
@@ -16,23 +17,24 @@ func main() {
 	cfg := config.Data{}
 	err := cfg.Load()
 	if err != nil {
-		fmt.Println(err.Error())
-		os.Exit(1)
+		flag.Usage()
+		fmt.Printf("\n%s\n", err.Error())
+		os.Exit(config.TimeWithExitCode(1))
 	}
 	if config.Debug {
 		fmt.Printf("Reading API specification from \"%s\"\n", cfg.GetOpenAPIFilepath())
 	}
 	apiSpec, err := openapi.LoadAPISpec(cfg.GetOpenAPIFilepath())
 	if err != nil {
 		fmt.Println(err.Error())
-		os.Exit(1)
+		os.Exit(config.TimeWithExitCode(1))
```

Comment:
Consider writing an exit function that encapsulates this.
so we can call `Exit(1)` here and then that will call os.Exit.
Otherwise we have to keep remembering to add `config.TimeWithExitCode()` everywhere

Reply:
agreed. that makes more sense.
Shall we agree to only `Exit()` within main?

Reply:
makes sense.